frontend/src/features/comparisons/Comparison.tsx

```diff
@@ -143,12 +147,20 @@ const Comparison = ({
       let autoUidA = null;
       let autoUidB = null;
 
-      if (!uidA && autoFillSelectorA) {
-        autoUidA = await nextSuggestion(uidA, uidB, pollName);
+      if (uidA) {
+        selectorAHistory.push(pollName, uidA);
+      } else if (autoFillSelectorA) {
+        autoUidA = await nextSuggestion(pollName, [uidA, uidB], 'A');
       }
```

Comment:
I wonder if it could be simplified by moving the suggestion logic into `UidHistory` to get something like:
```ts
      if (uidA) {
        selectorAHistory.push(pollName, uidA);
      } else if (autoFillSelectorA) {
        autoUidA = await selectorAHistory.getSuggestion(pollName, {exclude: [uidB]})
      }
```